staging/src/k8s.io/client-go/tools/remotecommand/remotecommand.go

```diff
@@ -48,6 +49,8 @@ type Executor interface {
 	// is set, the stderr stream is not used (raw TTY manages stdout and stderr over the
 	// stdout stream).
 	Stream(options StreamOptions) error
+	// StreamContext initiates the transport of the standard shell streams through Context.
+	StreamContext(ctx context.Context, options StreamOptions) error
```

Comment:
this is breaking change, please send an email to kubernetes-sig-api-machinery@googlegroups.com to check if it's acceptable.

Reply:
Thanks for the tip, I have sent the email

Reply:
can you do a search to see if there are other implementations of this? If not it is probably fine to make this change.